integration-tests/pom.xml

```diff
@@ -66,6 +66,7 @@
         <module>scala</module>
         <module>kotlin</module>
         <module>mongodb-panache</module>
+        <module>narayana-stm</module>
```

Comment:
We need to add the execution of the native tests in the root `azure-pipelines.yml` file. You have a lot of existing examples in the file.

Reply:
+1